### PR TITLE
added link to custom icon setup

### PR DESCRIPTION
**Pull request by alexvoss**
As discussed on Mastodon, I added a link from the icons & emojis documentation to the place in the setup configuration that describes how to add custom icons. Have a great day!

### CODE REVIEW

File: docs/reference/icons-emojis.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ icon: material/emoticon-happy-outline
 
 One of the best features of Material for MkDocs is the possibility to use [more
 than 10,000 icons][icon search] and thousands of emojis in your project 
-documentation with practically zero additional effort. Moreover, custom icons 
-can be added and used in `mkdocs.yml`, documents and templates.
+documentation with practically zero additional effort. Moreover, [custom icons 
+can be added](../../setup/changing-the-logo-and-icons/#additional-icons) and
```
Comment (squidfunk) [inline docs/reference/icons-emojis.md:10]: Could you please move the link definition outside of the text? We define all links outside, so the text is better to read (as can be seen in line 13) 🚀

**Review by squidfunk — CHANGES_REQUESTED** [review]
Thanks for the PR! Only some minor cosmetic stuff.

File: docs/reference/icons-emojis.md (hunk 1 above)
Comment (alexvoss) [reply docs/reference/icons-emojis.md]: Ok, done.

**Comment by squidfunk** [comment]
Perfect, thanks!

**Review by squidfunk — APPROVED** [review]

**Pull request merged**